rfc000-rfc-process.md

```diff
@@ -124,6 +136,7 @@ RFC: unassigned
 Author: Alan Smithee <asmithee@example.com>
 Status: Draft
 Type: <Standards Track, Informational, Process>
+<Replaces: RFCxxx>
```

Comment:
I'm looking for a establishing a backwards link here. Without something like this, it's hard to know that this RFC has a relationship with an older one. For some time cases, it's likely you'll find the newer one but want the older one, like if we just created a newer version of an API, but someone is working on software that used the older version.

I was trying to infer that the metadata line was optional with the <> characters, since the above line infers that you should pick one of the above. Using [] characters instead might make more since to those who are used to that syntax.